templates/publisher/publicise.html

```diff
@@ -0,0 +1,156 @@
+          </div>
+          <div class="col-8">
+            <pre><code>[![{{ data.text }}](https://snapcraft.io/static/images/badges/{{ lang }}/snap-store-black.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
+          </div>
+        </div>
+
+        <div class="row">
+          <hr />
+        </div>
+
+        <div class="row">
+          <div class="col-2">
+          </div>
+          <div class="col-8">
+            <div class="col-5">
+              <p class="snapcraft-publicise__images">
+                {% set url = "images/badges/" + lang + "/snap-store-white.svg" %}
+                <a href="https://snapcraft.io/{{ snap_name }}" title="{{ data.text }}">
+                  <img src="{{ static_url(url) }}" />
+                </a>
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-2">
+            <label>HTML:</label>
+          </div>
+          <div class="col-8">
+            <pre><code>&lt;a href="https://snapcraft.io/{{ snap_name }}" title="{{ data.text }}"&gt;
+  &lt;img src="https://snapcraft.io/static/images/badges/{{ lang }}/snap-store-white.svg" alt="" /&gt;
+&lt;/a&gt;</code></pre>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-2">
+            <label>Markdown:</label>
+          </div>
+          <div class="col-8">
+            <pre><code>[![{{ data.text }}](https://snapcraft.io/static/images/badges/{{ lang }}/snap-store-white.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
+          </div>
+        </div>
+      </div>
+    {% endfor %}
+
+    <div class="row">
+      <hr />
+    </div>
+
+    <div class="row">
+      <div class="col-2">
+        Download all:
+      </div>
+      <div class="col-8">
+        <a href="https://github.com/snapcore/snap-store-badges/archive/v1.zip" class="p-button">zip</a>
+        <a href="https://github.com/snapcore/snap-store-badges/archive/v1.tar.gz" class="p-button">tar.gz</a>
+        <a href="https://raw.githubusercontent.com/snapcore/snap-store-badges/master/LICENSE.md" target="_blank" class="p-link--external">View license</a>
+      </div>
+    </div>
+  </div>
+</div>
+{% endblock %}
+
+{% block scripts %}
+<script>
+  Raven.context(function () {
+    var languagePicker = document.querySelector('.js-language-select');
+
+    function showLanguage(language) {
+      var open = document.querySelector("#" + language + "_content");
+
+      document.querySelector(".js-language-content:not(.u-hide)").classList.add("u-hide");
+      open.classList.remove("u-hide");
+    }
+
+    var checked = document.querySelector("[name='language']:checked");
+
+    if (!checked) {
+      checked = document.querySelector("[name='language']");
+      checked.setAttribute("checked", "checked");
+    }
+
+    showLanguage(checked.value);
+
+    languagePicker.addEventListener("change", function(e) {
+      console.log(this.elements["language"].value);
```

Comment:
That's why we should avoid inline scripts - `console` got through as it was not linted.

Could we get this script into small module that would initialize this page?

Reply:
A user will be in the publisher part of the site anyway, so I've added it to the publisher module.